dm/syncer/checkpoint.go

```diff
@@ -683,6 +817,13 @@ func (cp *RemoteCheckPoint) CheckGlobalPoint() bool {
 	return time.Since(cp.globalPointSaveTime) >= time.Duration(cp.cfg.CheckpointFlushInterval)*time.Second
 }
 
+// CheckLastSnapshotCreationTime implements CheckPoint.CheckLastSnapshotCreationTime.
+func (cp *RemoteCheckPoint) CheckLastSnapshotCreationTime() bool {
+	cp.RLock()
+	defer cp.RUnlock()
+	return time.Since(cp.lastSnapshotCreationTime) >= time.Duration(cp.cfg.CheckpointFlushInterval)*time.Second
```

Comment:
After supporting async flush checkpoint, can we reduce the default value of CheckpointFlushInterval?

Reply:
that sounds good, will do a further testing to see if such interval could impact perf